authors/apps/articles/tests/test_views.py

```diff
@@ -0,0 +1,122 @@
+# articles/tests/test_views.py
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework.test import force_authenticate
+
+from authors.apps.authentication.models import User
+from authors.apps.articles.views import ArticleList, ArticleDetail
+
+
+class AuthenticationTests(APITestCase):
+    def setUp(self):
+        """
+        Data for the tests
+        """
+        self.data = {
+
+            "title": "Be a python coder in three weeks without a hassle",
+            "description": "Are you ready?",
+            "body": "It takes grit",
+            "tagList": ["javscript", "python"],
+            "images": ["image1", "image2"]
+
+        }
+        # Set up the registration url.
+        self.articles_url = reverse('articles:all_articles')
+        self.request_factory = APIRequestFactory()
+        User.objects.create(username='olivia', email='olivia@gmail.com', password='1232444nm')
+
+    def test_create_without_login(self):
+        """
+        Test that a user can't create an article without authorisation.
+        """
+        response = self.client.post('/api/articles/', self.data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_create_article_successfully(self):
+        """
+        Test that user can't create article without data
+
```

Comment:
Is this extra whitespace in the docstrings necessary?

Reply:
White spaces removed @lennykioko